pkg/crypto/crypto.go

```diff
@@ -1145,8 +1136,8 @@ func NewClientCertificateTemplateForDuration(subject pkix.Name, lifetime time.Du
 	}
 }
 
-func warnAboutCertificateLifeTime(name string, defaultLifetimeInDays int) {
-	defaultLifetimeInYears := defaultLifetimeInDays / 365
+func warnAboutCertificateLifeTime(name string, defaultLifetimeDuration time.Duration) {
+	defaultLifetimeInYears := defaultLifetimeDuration / 365 / 24
```

Comment:
should we change this to log a more granular value rather than years?

Reply:
So far certificate issued for month is standard practice, but later we'll want to shorten those. Years is okay now